src/azure-openapi-linter/tests/resources/ArraySchemaWithoutItems.json

```diff
@@ -57,10 +57,7 @@
             "description": "the models to return",
             "properties": {
                 "mymodels": {
-                    "type": "array",
-                    "items": {
-                        "$ref": "#/definitions/Model"
-                    }
+                    "type": "array"
```

Comment:
if this was incorrect, how was the test passing? why were we not catching the fact that the spec was not reporting a violation?